bookshelf/src/app.php

```diff
@@ -41,7 +41,8 @@
 // turn debug on by default
 $app['debug'] = !in_array(
     getenv('BOOKSHELF_DEBUG'),
-    ['false', '', '0', 'off', 'no']
+    ['false', '', '0', 'off', 'no'],
+    true
```

Comment:
In my opinion, the value of `$app['debug']` must be by default `false`. `debug=true` is a dangerous configuration on prod because it may reveal your secrets (db password etc).